docs/guide/getting-started/installation.md

```diff
@@ -2,24 +2,36 @@
 
 You can install NuxtAuth using nuxi:
 
-```bash
-npx nuxi@latest module add sidebase-auth
+::: code-group
+
+```bash [npm]
+npx nuxi module add sidebase-auth
+```
+
+```bash [pnpm]
+pnpm exec nuxi module add sidebase-auth
 ```
 
+```bash [yarn]
+yarn dlx nuxi module add sidebase-auth
+```
+
+:::
+
 ::: details Manual installation
 
 ::: code-group
 
 ```bash [npm]
-npm i -D @sidebase/nuxt-auth
+npm i @sidebase/nuxt-auth
 ```
 
 ```bash [pnpm]
-pnpm i -D @sidebase/nuxt-auth
+pnpm i @sidebase/nuxt-auth
 ```
 
 ```bash [yarn]
-yarn add --dev @sidebase/nuxt-auth
+yarn add @sidebase/nuxt-auth
```

Comment:
Please revert to the dev dependency so we can accept your other adjustment :)

Additionally - does the `nuxi module add` command work for you? There's https://github.com/sidebase/nuxt-auth/issues/932 which reported it not functioning

Reply:
I reverted to dev dep. The `nuxi module add sidebase-auth` command works fine, although it installed an RC Version of `@sidebase/nuxt-auth` for me. I could submit an PR regarding that. It also installs it as a regular dependency, and it installs `next-auth` too. Which is not mentioned in the docs. Do we want to include it in the docs too?

Reply:
The `rc` installation was a small hiccup from our side which we fixed today, it should no longer be installed as `latest`, thanks for noticing it as well!
`next-auth` is a peer dep, it would very much depend on your package manager

Reply:
I use pnpm, and it did not install `next-auth' and I ran into some issues. Don't you think that it should be included in the docs anyway?